src/structures/contexts/DiscordMessageContext.ts

```diff
@@ -8,16 +8,15 @@ import Command from "../../groups/Command.js";
 
 export default class DiscordMessageContext extends MessageContext {
 	private _splitArgs: string[];
-	private message: Message;
 	public command?: Command;
-	public author: User;
-	public guild?: Guild;
 
-	constructor(client: Client, message: Message, author: User, guild?: Guild) {
+	constructor(client: Client, 
```

Comment:
`client`'s access specifier should be explicited, and the parameter should be placed after a line break.
```suggestion
	constructor(
		public client: Client, 
```